tests/features/capsulahub-application.feature

```diff
@@ -0,0 +1,103 @@
+Feature: Build and Run commands for CapsulaHub CLI
+
+  Background:
+    Given an npm package called @capsulajs/capsula-hub
+    And   a default port 55555
+    And   a default configProvider httpFile
+    And   the following <configProvider> available
+          |<configProvider>                   |
+          |LocalFileConfigurationProvider     |
+          |HttpFileConfigurationProvider      |
+          |HardcoreServerConfigurationProvider|
```

Comment:
After the latest changes in ConfigurationService we should change **HardcoreServerConfigurationProvider** to **ScalecubeConfigurationService** and **hardcoreServer** to **scalecube**